docs/proposals/19-monitoring.md

```diff
@@ -0,0 +1,373 @@
+---
```

Comment:
Can the filename be a more descriptive one - for example `19-migrating-to-prometheus-operator.md`?